bin/migration-scripts/lts-to-lts/0.16-to-0.23/playbooks/tasks/upgrade.yaml

```diff
@@ -282,8 +281,10 @@
 
 - name: "(5/6) Upgrade Astronomer: Helm upgrade - Update Airflow configurations. Watch for Airflow charts to upgrade version with 'helm list --all-namespaces', watch for airflow pods to start back up."
   shell: |
```

Comment:
We should switch these shell entries to the [block scalar style](https://yaml-multiline.info) which is much easier to read. We switched to this in the newer migrations. EG: https://github.com/astronomer/astronomer/blob/b7305cc59108ca7/migrations/scripts/lts-to-lts/0.23-to-0.25/playbooks/tasks/upgrade.yaml

Reply:
Like that change quite a bit.